### DISCUSSION TITLE
Automatically changes code review to Approved when PR is merged.

### DISCUSSION THREAD

**Discussion by awxiaoxian2020** [community/community#19527] [Pull Requests] [answered]
Or change button to `Approved and Merged`. In some cases, we foget approve the pr and directly merged. we should keep consistency.

**Comment by gtirloni** [comment, marked as answer]
IMHO, this is not a good idea. Approval means a human look at the code, reviewed it and agreed to the changes.

If someone just merges a PR without getting a explicit approval, that doesn't mean anything. Nobody reviewed and approved the changes.

Are you suggesting this because of some corporate policy that requires all PRs to be approved? That's my only guess. If so, add a protected branch rule and codeowners file.

**Reply by awxiaoxian2020** [reply]
Thanks very much indeed.
